src/graph/impl/KokkosGraph_Distance2Color_impl.hpp

```diff
@@ -466,7 +477,7 @@ class GraphColorDistance2
               }
             }
           }
-          else
+          else if (colors(v) == 0 || colors(v) == CONFLICTED || colors(v) == UNCOLORABLE)
```

Comment:
@brian-kelley  why are we checking colors(v) UNCOLORABLE before actually setting it to UNCOLORABLE in the next line?

Reply:
That part of the loop condition is redundant, yeah.